README.md

```diff
@@ -1,70 +1,71 @@
-# ![Base16](logo.png)
+<!-- # ![Base16](logo.png) -->
 
-An architecture for crafting color schemes and easily translating them to
-your favorite apps - based on carefully chosen syntax highlighting using a
-base of sixteen colors.
+# BaseEverywhere
 
-It allows users to choose a color scheme they know will be available for all
-their applications (both GUIs and terminal applications).
+Smart build tooling for crafting high fidelity color schemes and easily translating them to all your favorite apps.  Originally based on the amazing work of [Chris Kempson](https://github.com/chriskempson/) and the [Base16 project](https://github.com/chriskempson/base16).
 
-It allows color scheme designers to select a palette of 16 colors once and then instantly generate support for many different applications.
 
-See the [List of supported programs](#template-repositories) on how to apply
-base16 schemes on your favorate program.
 
-## Gallery
+**Features**
+
+- Seamless builder support for _multiple_ style guides/architectures ([Base16](), [Base24](), etc.)
+- Over 230 beautiful and ready-to-use color schemes. [View the Gallery](https://base16-project.github.io/base16-gallery).
+- Over 70 supported GUI and terminal applications. [See the full list](#supported-applications).
+- Allows end users to choose a color scheme they know will be available _everywhere_.
+- Allows color scheme designers to craft a palette of colors once, instantly supporting for many different apps.
 
-We currently host 200+ color palletes. You can check out them at [Base16 Gallery](https://base16-project.github.io/base16-gallery).
 
 ## Documentation and Specification
 
-Base16 provides a set of guidelines detailing how to style syntax and how to
-code a _builder_ for compiling Base16 _schemes_ and _templates_.
+The _builder_ and _styling_ specs detail how to compile _schemes_ and _templates_ into application specific configurations.
+
 
 ### Scheme
 
-A scheme is a list of 16 colors to be used as a base to build application specific files. To make
-base16 look consistent, each color will be used for similar purpose in different
-programs. See [Styling Guidelines](/styling.md) for more information on how each
-color should be used.
+A scheme is simply a fixed palette of named colors.
```

Comment:
Just a suggestion, not requesting this. I feel like the use of "simply" adds unnecessary information.

```suggestion
A scheme is a fixed palette of named colors.
```